build.gradle

```diff
@@ -20,45 +19,16 @@ buildscript {
 }
 
 plugins {
-    id 'com.diffplug.spotless' version '6.25.0'
-    id 'com.github.ben-manes.versions' version '0.51.0'
+    id("eclipse")
+    id("com.diffplug.spotless") version "6.25.0"
+    id("com.github.ben-manes.versions") version "0.51.0"
 }
 
 apply from: "$rootDir/gradle/shipkit.gradle"
-apply from: "$rootDir/gradle/root/ide.gradle"
 apply from: "$rootDir/gradle/license.gradle"
 
 subprojects { proj ->
-    repositories {
-        mavenCentral()
-        google()
-    }
     plugins.withId('java') {
-        proj.apply from: "$rootDir/gradle/errorprone.gradle"
-        proj.apply from: "$rootDir/gradle/jfr.gradle"
         proj.apply from: "$rootDir/gradle/spotless.gradle"
```

Comment:
Shouldn't we apply this in `buildSrc/src/main/kotlin/mockito.java-conventions.gradle.kts` instead? That's the convention we should apply to all Java projects, so I think it should apply `spotless` as well. That way, we git rid of the `subprojects` call as well (which is discouraged because of performance implications)

Reply:
Yes that was my plan, but I ran into some issue with spotless, so I'd rather do it in another [task](https://github.com/orgs/mockito/projects/1/views/1?pane=issue&itemId=83037086).